test/e2e/search.test.js

```diff
@@ -105,4 +105,23 @@ describe('Search Plugin Tests', function() {
     await page.fill('input[type=search]', 'test');
     await expect(page).toEqualText('.results-panel h2', 'Test Page');
   });
+
+  test('search ignore diacritical marks', async () => {
+    const docsifyInitConfig = {
+      markdown: {
+        homepage: `
+          # Qué es
+
+          docsify genera su sitio web de documentación sobre la marcha. A diferencia de GitBook, no genera archivos estáticos html. En cambio, carga y analiza de forma inteligente sus archivos de Markdown y los muestra como sitio web. Todo lo que necesita hacer es crear un index.html para comenzar y desplegarlo en GitHub Pages.
+        `,
+      },
+      scriptURLs: ['/lib/plugins/search.min.js'],
+    };
+    await docsifyInit(docsifyInitConfig);
+    await page.fill('input[type=search]', 'documentacion');
+    await expect(page).toEqualText('.results-panel h2', 'Que es');
```

Comment:
is it possible to show the accent in the result ?

sorry if I missed any previous discussion regarding this. LMK

Reply:
I don't have a good idea how to do it...

Reply:
Looks like @sy-records got 2 of the 3 I listed. Perhaps we should open a new issue regarding missing diacritical marks in the search results so we can merge and release this PR.

Reply:
done. #1491

Reply:
Created a new issue for missing diacritical remarks in search results (#1492).